Assets/Scripts/FeedingSystem/DogFoodBowl.cs

```diff
@@ -80,17 +82,18 @@ int calculateDogFoodNeeded()
         return dataController.DogCount - dataController.ScoutingDogs.Count;
     }
 
-    public void FeedDogs()
+    public void FeedDogs(int foodType)
```

Comment:
If we're using the `DogFoodType` enum, feel as though the function should just take the enum not an int as the argument